### DISCUSSION TITLE
how to change output sample_rate

### DISCUSSION THREAD

**Discussion by forthcoming** [coqui-ai/TTS#3650] [General]
```python
tts = TTS('tts_models/multilingual/multi-dataset/xtts_v2').to('cpu')
model: Xtts = tts.synthesizer.tts_model
```

the default output sample_rate is 24000,how to change it to 16000? thanks

**Comment by erew123** [comment]
Try the config.json thats alongside wherever you have the model store. There is an output_sample_rate (or something close to that) that may work (reload the model of course).
